app/javascript/react/components/BrandShowContainer.js

```diff
@@ -31,20 +32,20 @@ const BrandShowContainer = (props) => {
         <div class='cell small-4 pizzaCard'>
           <div class='card'>
             <div class='card-section text-center'  key={pizza.id}>
-              {pizza.product_name}
+            <Link to={`/pizza/${pizza.id}`}>{pizza.product_name}</Link>
             </div>
           </div>
         </div>
       )
     })
     
     return(
-    <div>
+    <>
       <div className='cell small-11 text-center'>
         <h2>{brand.name}</h2>
       </div>
         {pizzaTiles}
```

Comment:
I would extract this info into a pizzaTile component to extract your html in lines 32-38